website/static/js/osfHelpers.js

```diff
@@ -701,7 +701,22 @@ var confirmDangerousAction = function (options) {
 
     bootbox.dialog(bootboxOptions);
 };
-
+/**
+ * Maps an object to an array of {key: KEY, value: VALUE} pairs
+ *
+ * @param {Object} obj
+ * @returns {Array} array of key, value pairs
+ **/
+var iterObject = function(obj) {
```

Comment:
This PR introduces a new helper function. But I don't see it used anywhere other than unit tests...?